cmd/service-controller/sites.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/skupperproject/skupper/client"
+)
+
+const (
+	SiteManagement string = "SiteManagement"
+)
+
+type SiteManager struct {
```

Comment:
As we already have some site related data served up, I would prefer to call this something else. Ideally more explicitly tied to the revoking of access, e.g. name the file revokeaccess.go, call this type AccessRevoker or something along those lines.

Reply:
Done